doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,33 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location in the etcd key-value store. Because of that, you do not run into
+merge conflicts as you often do with Git commits when you try to merge
+your `.git` history with the master copy of the repo. With large datatsets
+resolving a merge conflict might not be possible.
+
+A Pachyderm repository is the first entity that you configure to create
+a standard pipeline. You can create a repository by running the
+`pachctl create repo <name>` command or by using the Pachyderm UI. After
+creating the repository, you can add your data by using the
+`pachctl put file <repo@branch:/path><path-to-filesystem>` command. The
+path to the filesystem can be a local directory, file, or a URL.
```

Comment:
> The path to the filesystem can be a local directory, file, or a URL.

This is covered in other places where we talk about put file. No need to have it here. 

> This repository is called the *input repo* and is stored under
`pfs/<repo-name>`. All the input data for your pipeline is stored in
this repository.

I see what you're going for here, but this isn't quite right. Instead:

"Repo can be **input repos**, meaning users or external applications outside of Pachyderm are adding data to them or they can be **output repos** which are automatically created and written to by Pachyderm pipelines only (see LINK to pipelines.md for more info on pipelines)."